packages/ndla-ui/src/TreeStructure/FolderNameInput.tsx

```diff
@@ -38,23 +39,31 @@ const Row = styled.div`
   padding-right: ${spacing.xsmall};
 `;
 
-const InputWrapper = styled.div<{ loading?: boolean }>`
+const InputWrapper = styled.div<{ level: number }>`
   display: flex;
   margin: ${spacing.xxsmall} 0;
+  margin-left: ${({ level }) => 0.75 * spacingUnit * level + 2 * spacingUnit}px;
```

Comment:
Dette er et regnestykke som fort kan gå gærent. Er det mulig å legge inn litt parenteser her for å redusere sjansen for feil?

Reply:
Prettier nekter :/